src/java/org/apache/cassandra/service/StartupChecks.java

```diff
@@ -187,6 +190,58 @@ public void verify(StartupChecksOptions options) throws StartupException
         }
     }
 
+    // https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1057843
+    public static final StartupCheck checkKernelBug1057843 = new StartupCheck()
+    {
+        private final Range<CassandraVersion> affectedKernels = Range.closedOpen(
+        new CassandraVersion("6.1.64").toBaseVersion(), new CassandraVersion("6.1.66").toBaseVersion());
```

Comment:
yeah, I know it is confusing. I just wanted to avoid creating a duplicate of `CassandraVersion`. Some alternative is to use `Semver` but so far we have this library included only in tests. It shouldn't be a problem to include it in production code as its license is acceptable.